core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+
+- name: Download and unpack Helm binary
+  unarchive: 
+    src: "{{ apache_epirepo_path }}/files/helm-{{ helm_version }}-linux-amd64.tar.gz"
+    dest: "/usr/local/bin"
+    extra_opts:
+      - --strip=1
+      - --wildcards
+      - '*/helm'
+    remote_src: yes
+
+- name: Create Helm binary file symlink (RHEL) 
+  file:
+    src: "/usr/local/bin/helm"
+    dest: /usr/local/sbin/helm
+    state: link
+
+- name: Get list of Helm repos 
+  become: yes
+  shell: 'helm repo list | grep {{ helm_chart_repo_name }} || :'
+  register: helm_repo
```

Comment:
How about error support? What if repository won't be found and the rc will be 1?

Reply:
it always gives 0 code:
```
helm repo list | grep nonexisting  || :
echo $?
0
```

Reply:
But what if helm repolist will return error? How about using PIPESTATUS?

Reply:
added pipefail set